sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -314,4 +323,28 @@ private PlaySourceInternal translatePlaySourceToPlaySourceInternal(PlaySource pl
         }
         return playSourceInternal;
     }
+
+    private List<RecognizeChoice> translateRecognizeChoiceListtoInternal(List<com.azure.communication.callautomation.models.RecognizeChoice> recognizeChoices) {
+        return recognizeChoices.stream()
+            .map(e -> translateRecognizeChoiceToInternal(e))
+            .collect(Collectors.toList());
+    }
+
+    private RecognizeChoice translateRecognizeChoiceToInternal(com.azure.communication.callautomation.models.RecognizeChoice recognizeChoice) {
+        RecognizeChoice internalRecognizeChoice = new RecognizeChoice();
+        if (recognizeChoice.getLabel() != null) {
+            internalRecognizeChoice.setLabel(recognizeChoice.getLabel());
+        }
+        if (recognizeChoice.getPhrases() != null) {
+            internalRecognizeChoice.setPhrases(recognizeChoice.getPhrases());
+        }
+        if (recognizeChoice.getTone() != null) {
+            internalRecognizeChoice.setTone(translateDtmfToneToInternal(recognizeChoice.getTone()));
+        }
+        return internalRecognizeChoice;
+    }
+
+    private DtmfTone translateDtmfToneToInternal(com.azure.communication.callautomation.models.DtmfTone dtmfTone) {
```

Comment:
DtmfToneInternal

Reply:
do you mean the function name?

Reply:
the parameter name should be DtmfToneInternal, just as I mentioned the recognize ones.